src/router/routes.ts

```diff
@@ -211,6 +211,61 @@ export const routes = [
       filingType: FilingTypes.RESTORATION
     }
   },
+  {
+    path: '/amalg-reg-information',
+    name: RouteNames.AMALG_REG_INFORMATION,
+    component: Views.AmalgRegInformation,
+    meta: {
+      step: 1,
+      requiresAuth: true,
+      filingType: FilingTypes.AMALGAMATION,
+      filingSubType: AmalgamationTypes.REGULAR
+    }
+  },
+  {
+    path: '/amalg-reg-business-info',
+    name: RouteNames.AMALG_REG_BUSINESS_INFO,
+    component: Views.AmalgRegBusinessInfo,
+    meta: {
+      step: 2,
+      requiresAuth: true,
+      filingType: FilingTypes.AMALGAMATION,
+      filingSubType: AmalgamationTypes.REGULAR
+    }
+  },
+  {
+    path: '/amalg-reg-people-roles',
+    name: RouteNames.AMALG_REG_PEOPLE_ROLES,
+    component: Views.AmalgRegPeopleRoles,
+    meta: {
+      step: 3,
+      requiresAuth: true,
+      filingType: FilingTypes.AMALGAMATION,
+      filingSubType: AmalgamationTypes.REGULAR
+    }
+  },
+  {
+    path: '/amalg-reg-share-structure',
+    name: RouteNames.AMALG_REG_SHARE_STRUCTURE,
+    component: Views.AmalgRegShareStructure,
+    meta: {
+      step: 4,
+      requiresAuth: true,
+      filingType: FilingTypes.AMALGAMATION,
+      filingSubType: AmalgamationTypes.REGULAR
+    }
+  },
+  {
+    path: '/amalg-reg-review-confirm',
+    name: RouteNames.AMALG_REG_REVIEW_CONFIRM,
+    component: Views.AmalgRegReviewConfirm,
+    meta: {
+      step: 5,
+      requiresAuth: true,
+      filingType: FilingTypes.AMALGAMATION,
+      filingSubType: AmalgamationTypes.REGULAR
+    }
+  },
```

Comment:
This provides a different route (in URL) for each step.

Each step is actually rendered here: https://github.com/bcgov/business-create-ui/blob/870f1ee1fdff9fe3a905d47004c95fc03822df39/src/App.vue#L181